apps/api/src/app/auth/usecases/register/user-register.usecase.ts

```diff
@@ -21,6 +21,8 @@ export class UserRegister {
   ) {}
 
   async execute(command: UserRegisterCommand) {
+    if (process.env.DISABLE_USER_REGISTRATION !== 'false') throw new ApiException('Account creation is disabled');
```

Comment:
```suggestion
    if (process.env.DISABLE_USER_REGISTRATION === 'true') throw new ApiException('Account creation is disabled');
```

I would go here with the more explicit check, in case some users from previous version haven't updated the .env file they have to add the "false" attribute